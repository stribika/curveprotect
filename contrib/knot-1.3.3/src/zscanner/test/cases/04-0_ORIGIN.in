$TTL	1

; OK
$ORIGIN .		; Root domain
@	NS	@	; Use origin
a.	NS	@	; Absolute dname
a	NS	@	; Relative dname
$ORIGIN tld.		; 1. level domain
@	NS	@	; Use origin
a.	NS	@	; Absolute dname
a	NS	@	; Relative dname
$ORIGIN second.tld.	; 2. level domain
@	NS	@	; Use origin
a.	NS	@	; Absolute dname
a	NS	@	; Relative dname
$ORIGIN \0320\ \\\"\.\@\*.tld.	; Label with special chars
@	NS	@	; Use origin
$ORIGIN b.a.9.8.7.6.5.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.8.b.d.0.1.0.0.2.ip6.arpa.	; IPv6 reverse
@	NS	@	; Use origin
$ORIGIN 12345678901234567890123456789012345678901234567890123456789012\051.tld.		; Label of maximal length
@	NS	@	; Use origin
$ORIGIN 123456789012345678901234567890123456789012345678901234567890123.123456789012345678901234567890123456789012345678901234567890123.123456789012345678901234567890123456789012345678901234567890123.1234567890123456789012345678901234567890123456789012345678901.	; Domain name of maximal length
@	NS	@	; Use origin
$origin .		; Lower-case
@	NS	@	; Use origin

; KO
$ORIGIN 		; Empty input
.	NS	.	; Is OK, but shouldn't be processed due to previous error stop!
