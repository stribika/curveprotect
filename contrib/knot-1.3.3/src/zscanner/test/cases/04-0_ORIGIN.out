OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=00
------
OWNER=016100
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=00
------
OWNER=016100
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=00
------
OWNER=03746C6400
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=03746C6400
------
OWNER=016100
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=03746C6400
------
OWNER=016103746C6400
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=03746C6400
------
OWNER=067365636F6E6403746C6400
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=067365636F6E6403746C6400
------
OWNER=016100
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=067365636F6E6403746C6400
------
OWNER=0161067365636F6E6403746C6400
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=067365636F6E6403746C6400
------
OWNER=082030205C222E402A03746C6400
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=082030205C222E402A03746C6400
------
OWNER=0162016101390138013701360135013001300130013001300130013001300130013001300130013001300130013001300138016201640130013101300130013203697036046172706100
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=0162016101390138013701360135013001300130013001300130013001300130013001300130013001300130013001300138016201640130013101300130013203697036046172706100
------
OWNER=3F31323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323303746C6400
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=3F31323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323303746C6400
------
OWNER=3F3132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333F3132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333F3132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333D3132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303100
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=3F3132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333F3132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333F3132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333D3132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303100
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=00
------
ERROR=ZSCANNER_EBAD_ORIGIN
------
